actix-http/src/response.rs

```diff
@@ -639,18 +639,10 @@ impl ResponseBuilder {
         self.body(Body::from_message(BodyStream::new(stream)))
     }
 
-    #[inline]
```

Comment:
keep the inline annotation

Reply:
The reason I "removed" it is because it never existed for `json2()`: I just renamed `json2()` to `json()` and removed the original `json()` function, and since `#[inline]` was never on `json2()`, "keeping" `#[inline]` would be more like adding it.

If you want me to inline the new `json()` I can, but I want to clarify that I'm not really removing it.

Reply:
Sorry you're right I read the diff wrong.